iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -254,6 +262,32 @@ public void commitAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable
     }
   }
 
+  @Override
+  public void rollbackAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTable, EnvironmentContext context)
+      throws MetaException {
+    context.getProperties().put(INITIALIZE_ROLLBACK_ALTER, "true");
+    this.catalogProperties = getCatalogProperties(hmsTable);
+    try {
+      this.icebergTable = Catalogs.loadTable(conf, catalogProperties);
+    } catch (NoSuchTableException nte) {
+      // iceberg table was not yet created, no need to delete the metadata dir separately
+      return;
+    }
+    // we want to keep the data files but get rid of the metadata directory
+    hmsTable.getParameters().put(InputFormatConfig.EXTERNAL_TABLE_PURGE, "FALSE");
```

Comment:
Will this change actually be saved into the HMS db? If so, what if the original table had this property as true? Should we change it silently here due to the rollback?

Reply:
No, it won't :). This some leftover code from the previous implementation, which I forgot to remove.